stack/hlsconstructs/batch.py

```diff
@@ -16,9 +17,9 @@ def __init__(
         instance_types: list,
         maxv_cpus: int,
         ssh_keyname: str,
-        efs: aws_efs.CfnFileSystem = None,
+        efs: aws_efs.CfnFileSystem | None = None,
```

Comment:
+1 to fixing the type, but it looks like in practice this can't be null? I'm seeing a couple lines like

```python
            user_data_string = str(userdata_file, "utf-8").format(
                efs.ref, cloudwatch_ssm_param
            )
```
where we're accessing `.ref`

Maybe suggest,
```suggestion
        efs: aws_efs.CfnFileSystem,
```
?

Reply:
Good catch!